nginx.conf

```diff
@@ -0,0 +1,43 @@
+
+error_log  logs/error.log  debug;
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+    #gzip  on;
+
+    # DataDome configuration
+    lua_ssl_verify_depth 3;
+    lua_ssl_trusted_certificate /etc/ssl/certs/ca-certificates.crt;
+
+    resolver 8.8.8.8;
+    init_by_lua_block {
+        require("datadome_openresty").setDatadomeConf( { datadome_server_side_key = "server_side_key", } )
+    }    
+
+    server {
+        listen       80;
+        server_name  localhost;
+
+         location / {
+            access_by_lua_block {
+                if ngx.req.is_internal() == false then
+                    require("datadome_openresty").validateRequest()
+                end
+            }
+	    
+            proxy_pass http://example.org;
```

Comment:
what is this used for ? 
It is misleading - i'd rather have a section commented out with a clear explanation - or removed
bare minimum config is good

Reply:
this is used to see a web page (example.org) or else we won't see anything.

Reply:
shouldn't it be https://point.datashield.co in this case ? :-P